src/context/Controller.js

```diff
@@ -0,0 +1,26 @@
+import React, { createContext, useState } from 'react';
+
+export const Context = createContext();
+export function Provider({ children }) {
+    const [values, setValues] = useState({});
+
+    function sendNewState(newState) {
+        var request = new XMLHttpRequest();
+            
+        request.open('POST', '/values', newState);
+        request.send();
+    }
+
+    function onControllerChange(name, value) {
+        const newState = {
+            ...values,
+            [name]: value,
+        };
+        sendNewState(newState);
+        setValues(newState);
```

Comment:
Do you think it would be a better indicator if you print to console?

Reply:
done